src/puppetlabs/puppetdb/http/query.clj

```diff
@@ -153,3 +211,88 @@
    :post [(are-queries-different? req %)]}
   (restrict-query ["=" "title" title]
                   req))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Conversion/validation of query parameters
+
+(defn parse-boolean
```

Comment:
That seems reasonable. Can we call them 'coerce-to-boolean', etc? That's how I thought of them at first, and how you described them.